share/instruments/instruments.xml

```diff
@@ -9967,6 +9989,85 @@
 <!--                  <clef>TAB</clef> -->
                   <genre>common</genre>
             </Instrument>
+            <Instrument id="5-string-electric-bass-high-c">
+                  <longName>5-str. Electric Bass (High C/Tenor)</longName>
```

Comment:
What about using this string for the trackname and make longname shorter ? "5-str. Electric Bass (High C/Tenor)" looks quite long on a score...

Reply:
Hope I got it right now